OracleSOA/dockerfiles/12.1.3/Dockerfile.developer

```diff
@@ -0,0 +1,95 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for SOA Suite 12.1.3 Quick Start Distro
+# 
+# IMPORTANT
+# ---------
+# The resulting image of this Dockerfile DOES NOT contain a SOA Suite Domain.
+# For that, look into the folder 'samples' for an example on how
+# to create a domain on a new inherited image.
+#
+# You can go into 'samples/12c-domain' after building the generic raw image
+# and build that image, for example:
+# 
+#   $ cd samples/12c-domain
+#   $ sudo docker build -t mysoa .
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_12.1.3.0.0_soaqs_Disk1_1of1.zip
+#     Download the Quick Start installer from http://www.oracle.com/technetwork/middleware/soasuite/downloads/index.html
+#
+# (2) jdk-7u75-linux-x64.rpm
+#     Download from http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html
+#
+# HOW TO BUILD THIS IMAGE
+# ------------------w-----
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ sudo docker build -t oracle/soa:12.1.3-dev . 
+#
+# Pull base image
+# ---------------
+FROM oraclelinux:7
+
+# Maintainer
+# ----------
+MAINTAINER Jorge Quilcate <quilcate.jorge@gmail.com>
+
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV JAVA_RPM jdk-7u75-linux-x64.rpm
```

Comment:
As above. I'm really not comfortable releasing stuff that relies on JDK7 which is no longer receiving public updates.